src/Sylius/Component/Core/Model/ProductVariant.php

```diff
@@ -184,6 +191,27 @@ public function setPrice($price)
     /**
      * {@inheritdoc}
      */
+    public function getOriginalPrice()
+    {
+        return $this->originalPrice;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setOriginalPrice($originalPrice)
+    {
+        if (!is_int($originalPrice)) {
+            throw new \InvalidArgumentException('Original price must be an integer.');
+        }
+        $this->originalPrice = $originalPrice;
+
+        return $this;
```

Comment:
We're trying to remove method chaining and fluent interfaces in places like setters in Sylius.